### PR TITLE
Community friendly rework

### PR DESCRIPTION
**Pull request by vferat**
- [x] Add Contributing
- [x] Add Citing
- [x] Add install

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,46 @@
 [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
-
+[![PyPI version](https://badge.fury.io/py/pycrostates.svg)](https://badge.fury.io/py/pycrostates)
```
Comment (mscheltienne) [inline README.md:2]:
```suggestion
[![PyPI version](https://badge.fury.io/py/pycrostates.svg)](https://badge.fury.io/py/pycrostates)
[![Conda Version](https://img.shields.io/conda/vn/conda-forge/pycrostates.svg)](https://anaconda.org/conda-forge/pycrostates)
```

File: README.md [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
+[![Documentation Status](https://readthedocs.org/projects/pycrostates/badge/?version=latest)](https://pycrostates.readthedocs.io/en/latest/?badge=latest)
```
Comment (mscheltienne) [inline README.md:3]:
```suggestion
```

Duplicated of badge below?

File: README.md [hunk 3]
```diff
@@ -1,39 +1,46 @@
...
 An open-source Python package for analyzing EEG microstates developed to work with [MNE-python](https://mne.tools/stable/index.html).
 
 ## Documentation
+[![Documentation Status](https://readthedocs.org/projects/pycrostates/badge/?version=latest)](https://pycrostates.readthedocs.io/en/latest/?badge=latest)
 
 The documentation including tutorials and example can be found [here](https://pycrostates.readthedocs.io/en/latest/).
 
+
 ## Installation
+![PyPI - Downloads](https://img.shields.io/pypi/dm/pycrostates)
```
Comment (mscheltienne) [inline README.md:20]: I am not getting the same numbers, and the link is only to the image?

Suggestion for Pypi:
`[Downloads](https://pepy.tech/project/pycrostates)`

Suggestion for conda:
`[Anaconda-Server Badge](https://anaconda.org/conda-forge/pycrostates)`

I don't find a download / month for `conda`.

File: README.md (hunk 2 above)
Comment (vferat) [reply README.md:4]: I don't think this is not really an issue. Feel free to delete it  before merging if you think it's better

**Comment by mscheltienne** [comment]
DOI is not rendering anymore:

Screenshot 2022-08-10 at 15 59 00

**Review by mscheltienne — APPROVED** [review]
LGTM, merge if happy. The render of the DOI badge fixed itself.

**Pull request merged**